test/integration/connect/envoy/case-centralconf/s1.hcl

```diff
@@ -1,16 +1,19 @@
 services {
   name = "s1"
   port = 8080
+
   connect {
     sidecar_service {
+      port = 21000
```

Comment:
Is it worth adding a comment here about why we are explicit about port in this case (and presumably not others)?